egs2/aishell/asr1/run_streaming.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
@ShigekiKarita and @kamo-naoyuki, I'm thinking of using a different directory from `asr1` for streaming, e.g., `asr2`, `asr1_strm`, or `sasr1`.
This is because the streaming model is significantly different from the off-line model.
We could also have some different stages (e.g., by using T-S training with the off-line model).

What do you think?

Reply:
`asr1_strm` sounds good.

Reply:
OK, I will make a new directory asr_strm.

Reply:
Same directory is fine for me.

This streaming mode can be activated only changing the encoder configuration.  `run_streaming.sh` is the minimal requirement and it's easy to use it in another recipe, but different directory is a little bit bothering to create each recipe for each streaming task.

Reply:
OK current implementation, yes, and let's keep the same directory name.
As I mentioned we may also use teacher-student training and different inference schemes in the future, but we can consider it at that timing.